go.mod

```diff
@@ -85,7 +83,8 @@ require (
 	k8s.io/kubectl v0.0.0
 	k8s.io/kubernetes v1.17.3
 	k8s.io/utils v0.0.0-20200229041039-0a110f9eb7ab // indirect
-	sigs.k8s.io/sig-storage-lib-external-provisioner v4.0.0+incompatible
+	sigs.k8s.io/sig-storage-lib-external-provisioner v4.0.0+incompatible // indirect
```

Comment:
I tried to run `go mod tidy` (and to remove it manually) but somehow I was not able to remove this import:

```bash
$ go mod tidy
go: finding module for package sigs.k8s.io/sig-storage-lib-external-provisioner/controller/metrics
go: found sigs.k8s.io/sig-storage-lib-external-provisioner/controller/metrics in sigs.k8s.io/sig-storage-lib-external-provisioner v4.1.0+incompatible
```

and

```bash
$ go mod why sigs.k8s.io/sig-storage-lib-external-provisioner/controller/metrics
# sigs.k8s.io/sig-storage-lib-external-provisioner/controller/metrics
k8s.io/minikube/pkg/storage
sigs.k8s.io/sig-storage-lib-external-provisioner/v5/controller
sigs.k8s.io/sig-storage-lib-external-provisioner/v5/controller.test
sigs.k8s.io/sig-storage-lib-external-provisioner/controller/metrics
```